src/configurations/destinations/gainsight_px/db-config.json

```diff
@@ -62,6 +65,8 @@
         "ketchConsentPurposes"
       ],
       "web": [
+        "dataCenter",
+        "useNativeSDK",
```

Comment:
I don't think we need useNativeSDK anymore as we now have connectionMode. We can remove this

Reply:
done